instance.go

```diff
@@ -7,10 +7,21 @@ import (
 
 // Instance hold information for mastodon instance.
 type Instance struct {
-	URI         string `json:"uri"`
-	Title       string `json:"title"`
-	Description string `json:"description"`
-	EMail       string `json:"email"`
+	URI         string            `json:"uri"`
+	Title       string            `json:"title"`
+	Description string            `json:"description"`
+	EMail       string            `json:"email"`
+	Version     string            `json:"version,omitempty"`
+	URLs        map[string]string `json:"urls,omitempty"`
+	Stats       InstanceStats     `json:"stats,omitempty"`
```

Comment:
I'm not sure but this should be `*InstanceStats`

Reply:
:ok_woman: b5c30f3